test/serverTest.ts

```diff
@@ -0,0 +1,54 @@
+import chai, { expect } from "chai";
+import chaiHttp from "chai-http";
+import request from "supertest";
+import { app } from '../src/server'
+
+chai.should();
+chai.use(chaiHttp);
+
+describe("Server API", () => {
+    
+    // Case where we have no fields or parameters specified
+    it("should GET /", async function() {
```

Comment:
We should put something more business oriented, like what we entered and what we want in output. So we could easily understand the purpose of the test
```suggestion
    it("should render all the fields with no param are passed", async function() {
```